runner/runners/bazel.go

```diff
@@ -34,7 +38,8 @@ func preProcessBazel(filer snapshot.Filer, cmd *runner.Command) error {
 		return fmt.Errorf("Nil ExecuteRequest data in Command with RunType Bazel")
 	}
 	if err := fetchBazelCommand(bzFiler, cmd); err != nil {
-		return fmt.Errorf("Error retrieving Bazel command data: %v", err)
+		// NB: Important to return this error as-is
```

Comment:
why?

Reply:
added comments here and in bazel/cas/service about returning NotFoundError. kind of an open question what the standard for errors should be - bubbling up vs wrapping in a new err, when to log, etc.